src/Error/PhpError.php

```diff
@@ -95,6 +94,10 @@ public function __construct(
         ?int $line = null,
         array $trace = []
     ) {
+        // E_STRICT is deprecated in PHP 8.4 and using it triggers a deprecation warning.
+        // phpcs:ignore
+        $this->levelMap[@E_STRICT] = 'strict';
```

Comment:
Should we guard this with a PHP version check? We can not have strict in PHP 8.4 if it is going away.

Reply:
If we want to just skip using E_STRICT in PHP 8.4 I can wrap this in a version check.

Reply:
I think that's a reasonable solution. If PHP effectively doesn't have E_STRICT in 8.4 we don't need to have support for ti.

Reply:
Done, only the issue of using `E_USER_ERROR` with `trigger_error()` remains now.